### ISSUE TITLE
Swiftlint configuration prevents swxftlint from finding the source files.

### ISSUE THREAD

**Issue by alimoeeny** [realm/SwiftLint#1990]
### Bug Report
Basically this swiftlint file prevents swxftlint to find the swift source files.
I believe this is the same problem, or a similar problem as was reported but never reproduced here: 
https://github.com/realm/SwiftLint/issues/514
##### Complete output when running SwiftLint, including the stack trace and command used

```zsh
$ swiftlint lint
Loading configuration from '.swiftlint.yml'
Linting Swift files in current working directory
No lintable files found at path ''
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? `0.24.0`
* Installation method used (Homebrew, CocoaPods, building from source, etc)? `brew
* Paste your configuration file:

```yml
included: # paths to include during linting. `--path` is ignored if present.
  - ../../../Assets
```

* Which Xcode version are you using (check `xcode-select -p`)? `Version 9.2 (9C40b)`

**Comment by marcelofabri** [comment]
Can you provide a sample folder/project where this can be reproduced? From the information you've added, it just looks like it's trying to lint only `../../../Assets` and there's nothing there.

Didn't you mean to use `excluded` instead of `excluded`? Also, using relative parent paths would be weird in that case. How does your directory structure look like and from where are you calling `swiftlint lint`?

**Comment by alimoeeny** [comment]
@marcelofabri so maybe the question is, what is the expected behavior if a path in the `included` is not found? 
I assumed it would just ignore it.
But if that is not what is intended maybe the error message can reflect that, does that make sense?

**Comment by alimoeeny** [comment]
btw, it is convenient (at least for me) to have includes that are not there be ignored.
I have some differences between my CI and my dev environment and keeping two separate but in sync copied of the config file are not very desirable.
